react_app/src/components/Game/ChallengeCard.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 
 import { GameContext } from "Context";
+import { openChallenge, closeChallenge } from "actions/game.actions"
```

Comment:
not using `closeChallenge`. your linter should be bitching about this though, you not been checking the messages?

Reply:
Fuck the linter... 

No, I have seen quite a few little bitchy messages from it about this...

Reply:
Resolved